### PR TITLE
Change filter expressions as per CIS Benchmark guide

### PR DESCRIPTION
**Pull request by rsgithub123**
Please refer to issue #9 - Change filter expressions as per CIS Benchmark guide ( https://d0.awsstatic.com/whitepapers/compliance/AWS_CIS_Foundations_Benchmark.pdf ). This PR is for changing 3 filters in alarms.tf as per CIS Banchmark guide.

Ensure a log metric filter and alarm exist for Management Console sign-in without MFA
Ensure a log metric filter and alarm exist for disabling or scheduled deletion of customer created CMKs
Ensure a log metric filter and alarm exist for AWS Config configuration changes

### CODE REVIEW

**Comment by mardinyadegar** [comment]
I have run the changes in my own personal AWS account and they all behave properly and as expected.

**Comment by reedloden** [comment]
@rsgithub123 Mind rebasing?

File: alarms.tf [hunk 1]
```diff
@@ -41,10 +41,10 @@ locals {
     "{ ($.eventName = CreateTrail) || ($.eventName = UpdateTrail) || ($.eventName = DeleteTrail) || ($.eventName = StartLogging) || ($.eventName = StopLogging) }",
     "{ ($.eventName = ConsoleLogin) && ($.errorMessage = \"Failed authentication\") }",
     "{ ($.eventName = DeleteGroupPolicy) || ($.eventName = DeleteRolePolicy) ||($.eventName=DeleteUserPolicy)||($.eventName=PutGroupPolicy)||($.eventName=PutRolePolicy)||($.eventName=PutUserPolicy)||($.eventName=CreatePolicy)||($.eventName=DeletePolicy)||($.eventName=CreatePolicyVersion)||($.eventName=DeletePolicyVersion)||($.eventName=AttachRolePolicy)||($.eventName=DetachRolePolicy)||($.eventName=AttachUserPolicy)||($.eventName=DetachUserPolicy)||($.eventName=AttachGroupPolicy)||($.eventName=DetachGroupPolicy)}",
-    "{ $.eventName = \"ConsoleLogin\" && $.additionalEventData.MFAUsed = \"No\" }",
+    "{ $.eventName = \"ConsoleLogin\" && $.additionalEventData.MFAUsed != \"Yes\" }",
```
Comment (reedloden) [inline alarms.tf]: This is missing two sets of parentheses.

`{ ($.eventName = \"ConsoleLogin\") && ($.additionalEventData.MFAUsed != \"Yes\") }",`

**Comment by aknysh** [comment]
@rsgithub123 can you please address the comment by @reedloden 
thanks everybody

**Comment by mardinyadegar** [comment]
Has this been resolved? Thanks for taking this further guys. Good work.

**Comment by mardinyadegar** [comment]
@aknysh, did this get resolved? I can make the requested changes if need be.

**Comment by mardinyadegar** [comment]
@reedloden funny to see you here =], maybe I will see you around the office some time too =]

Do you know if this has been complete? I am willing to help you out with fixing this one if it is not complete.

**Comment by reedloden** [comment]
We just ended up fixing this in our local fork. Would love to get this
fixed upstream, though. :-)

**Comment by mardinyadegar** [comment]
No problem @reedloden. Let me see if I can help with that. Hope to have an update soon :D

**Pull request merged**